### ISSUE TITLE
Some admin pages redirect to board index

### ISSUE THREAD

**Issue by vendethiel** [IntegraMOD/IntegraMOD151#65]
Not sure what that is about...

**Comment by IntegraMOD** [comment]
Is this still an issue after #60 fix?

**Comment by vendethiel** [comment]
The commit I made 9 days ago (that's linked here) wasn't fixed by #60. It used `&amp;` in places it shouldn't have and that broke things. I don't know if the other pages broke as well.

**Comment by IntegraMOD** [comment]
Im noticing in bare phpBB that the acp link goes to root index on the first try and requires a second click after passing acp login

**Comment by MWE001** [comment]
> Im noticing in bare phpBB that the acp link goes to root index on the first try and requires a second click after passing acp login

Yep. I hate that. It kind of annoys me to no end. Does v3 do it as well or is this a 2.x thing?

**Comment by vendethiel** [comment]
Do you guys have that with IM as well? It works fine on all premodded boards I have installed.
